test/test_tensor_spec.py

```diff
@@ -258,22 +259,24 @@ def test_multi_discrete(shape, ns):
     np.random.seed(0)
     ts = MultiDiscreteTensorSpec(ns)
     _real_shape = shape if shape is not None else []
-    _len_ns = [len(ns)] if len(ns) > 1 else []
+    nvec_shape = torch.tensor(ns).size()
+    if nvec_shape == torch.Size([1]):
```

Comment:
Can't we do
```
@pytest.mark.parametrize(
    "ns",
    [
        5,
        [5, 2, 3],
        [4, 5, 1, 3],
        [[1, 2], [3, 4]],
    ],
)
```
and avoid this?
```
torch.tensor(5)
```
should work just fine.

Reply:
Done, I also updated the __init__ method to take scalar for nvec